### PR TITLE
Renders react

### PR DESCRIPTION
**Pull request by CharlieAxelrod**
This code renders everything to the DOM; now I just need to style it!
@sharkhouse23 , could you take a look for me?

### CODE REVIEW

File: client/src/index.jsx [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+import About from './components/about.jsx';
+import More from './components/more.jsx';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      instructors: null
+      instructors: null,
+      courseId: 54
```
Comment (sharkhouse23) [inline client/src/index.jsx]: are you hard-coding the courseId right now to make sure it's rendering properly?

File: client/src/index.jsx [hunk 2]
```diff
@@ -15,21 +18,38 @@ class App extends React.Component {
...
     .then((data) => {
+      console.log(data);
       this.setState({
         instructors: data
       });
     });
   }
 
   render() {
+    console.log(this.state.instructors);
```
Comment (sharkhouse23) [inline client/src/index.jsx]: Consider commenting this out

File: server/index.js [hunk 3]
```diff
@@ -1,15 +1,39 @@
...
+            info.push(instructor);
+          });
+        promises.push(newPromise);
+      });
+      return Promise.all(promises)
+        .then(() => res.send(info));
+    });
 });
 
 app.listen(3000, () => {
```
Comment (sharkhouse23) [inline server/index.js]: where is the rest of this (ending brace, closing parens)?

Comment (sharkhouse23) [inline server/index.js]: @CharlieAxelrod made some minor comments. From a stylistic perspective, I think you did a great job separating your concerns and laying out the sub-components (especially with your More component). Made it easy to follow along and see how the different sub-components feed back into the App component.

From a functionality perspective, it was a little hard to follow your server requests due to the use of promises and  my unfamiliarity with sequelize. I wouldn't suggest re-factoring if everything is rendering properly for you though.

File: client/src/index.jsx (hunk 1 above)
Comment (CharlieAxelrod) [reply client/src/index.jsx]: Yeah, just for now

File: client/src/index.jsx (hunk 2 above)
Comment (CharlieAxelrod) [reply client/src/index.jsx]: Done!

File: server/index.js (hunk 3 above)
Comment (CharlieAxelrod) [reply server/index.js]: I think the line got cut off somewhere?

Comment (CharlieAxelrod) [reply server/index.js]: Thanks for the feedback. When I get a minute, I'll break the server code into functions to make it easier to follow.
